### ISSUE TITLE
Configure popup location (vsplit/split/...)

### ISSUE THREAD

**Issue by sotte** [TimUntersberger/neogit#315]
I would like to show my "popups" (commit, pull, push, etc) within a `vsplit`, not a `split`.

`neogit.open` already supports to set `kind` to configure the location of the opened feature buffer:
```
neogit.open({ kind = "split" })
neogit.open({ kind = "vsplit" })
```

`neogit.open` also allows you to open various popup buffers
```
neogit.open({"commit"})
neogit.open({"pull"})
```

But `neogit.open` but it does not allow you to combine this:
```
neogit.open({"commit", kind = "vsplit"})
```

**Describe the solution you'd like**
`neovim.open` should take the kind keyword into account.

Also: thank you so much for neogit. I've used fugitive for years, but neogit feels really good!

**Comment by sotte** [comment]
Implementing this feature would entail the following steps:

The `kinds` argument would have to passed to `popup.create(opts.kind)`:
https://github.com/TimUntersberger/neogit/blob/master/lua/neogit.lua#L29

Each popup file would have to forward the kind to `p:show()`
https://github.com/TimUntersberger/neogit/blob/master/lua/neogit/popups/diff.lua#L19

And `show` in `popup.lua` can then use the kind:
https://github.com/TimUntersberger/neogit/blob/master/lua/neogit/lib/popup.lua#L232

**Comment by sotte** [comment]
Closed with #316.
